package.json

```diff
@@ -106,7 +106,7 @@
     "react-native-gesture-handler": "~2.12.0",
     "react-native-get-random-values": "~1.9.0",
     "react-native-gifted-chat": "^2.0.1",
-    "react-native-maps": "1.7.1",
+    "react-native-maps": "^1.10.3",
```

Comment:
I'm getting this error. What is the node version on your device?  We are currently using version 16.13.0 for the project, but it seems that the `react-native-maps` package node needs a version higher than 18. So I can't test it. 

<img width="777" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/7f647d7a-c2dc-4fb4-af91-c2f6099ba479">

Reply:
if node needs to be updated, I think `.nvmrc` needs to be updated as well. As far as I can see, we have version `16.13.0` at the moment.

Which version of Node should we update to? Is `18.0.0` enough? Or do we want to use the latest version? As far as I know, the most current and stable version is `18.19.1`

Reply:
@d4v3d4vE can you answer here please.

Reply:
I would suggest du use the latest stable version, I'm on node `v22.0.0` and have no issues. The current LTS is `v20.12.2`

Reply:
[Expo says](https://docs.expo.dev/get-started/installation/):
> [Node.js LTS release](https://nodejs.org/en/) - Only Node.js LTS releases (even-numbered) are recommended.